Workbooks/Intune Audit/Audit/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"IntuneAuditActivity",
```

Comment:
You would probably want to name it like `Intune Audit Activity` since it will be appeared in a gallery as well as it will be localized.